zebra-rpc/build.rs

```diff
@@ -34,6 +39,14 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             .status()
             .expect("failed to clone external binary");
 
+        if ZALLET_COMMIT.is_some() {
+            let _ = Command::new("git")
+                .args(["checkout", ZALLET_COMMIT.unwrap()])
+                .current_dir(&build_dir)
+                .status()
+                .expect("failed to build external binary");
+        }
```

Comment:
It may be a good idea to use a specific commit here. there's already been a change in Zallet that breaks these tests (renamed config field, see comment below).

Nitpick:

```suggestion
        if let Some(zallet_commit) = ZALLET_COMMIT {
            let _ = Command::new("git")
                .args(["checkout", zallet_commit])
                .current_dir(&build_dir)
                .status()
                .expect("failed to build external binary");
        }
```